src/content/translations/ja/roadmap/index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: イーサリアムロードマップ
+description: イーサリアムのスケーラビリティ、セキュリティ、サステナビリティを改善する工程
+lang: ja
+template: roadmap
+image: ../../../../assets/roadmap/roadmap-main.png
+alt: "イーサリアムロードマップ"
+summaryPoints:
+buttons:
+  - label: 今後のアップグレード
+    toId: 予定されている変更
+  - label: 過去のアップグレード
+    to: /history/
+    variant: 概要
+---
+
+イーサリアムは、すでにグローバルで協調して稼働している強力なプラットフォームですが、さらなる改善が進められています。 大胆な改善案が提案されており、イーサリアムは現在の形から、フルスケールで最大の復元力を備えたプラットフォームへとアップグレードされる予定です。 これらの一連のアップグレードは、イーサリアムのロードマップに記載されています。
+
+**イーサリアムの過去のアップグレードについては、[履歴](/history/)のページをご覧ください。**
+
+## イーサリアムで予定されている変更点は何ですか？ {#what-changes-are-coming}
+
+将来のプロトコルの具体的な改善案については、イーサリアムのロードマップに概要が描かれています。 このロードマップは、イーサリアムのユーザーに次のメリットをもたらします。
+
+<CardGrid>
+  <RoadmapActionCard
+    to="/roadmap/scaling"
+    title="より安価なトランザクション"
+    image="scaling"
+    description="Rollups are too expensive and rely on centralized components, causing users to place too much trust in their operators. The roadmap includes fixes for both of these problems."
+    buttonText="More on reducing fees"
```

Comment:
Realizing these were not translated... @lukassim it looks like the `title` field above it being translated correctly... is there a way in Crowdin we can flag `description` and `buttonText` as fields that also need to be translated?